k8s/kubectl/admin/configmap/worker/sql/qserv-worker.sql

```diff
@@ -6,7 +6,7 @@ CREATE USER IF NOT EXISTS 'qsmaster'@'localhost';
 DROP DATABASE IF EXISTS qservw_worker;
 CREATE DATABASE qservw_worker;
 
-GRANT SELECT ON qservw_worker.* TO 'qsmaster'@'localhost';
+GRANT ALL ON qservw_worker.* TO 'qsmaster'@'localhost';
```

Comment:
I'm curious if the file is going to substitute the original one in package qserv. And if not then how are we going to keep in sync two parallel developments?

Reply:
I think in the future we should have schema in core/modules, next to all migration scripts. Another refactoring will be needed for that. We cannot have multiple versions of these files in different packages, that is not manageable.

Reply:
FYI all configuration files in qserv/admin/templates are shmux related. shmux procedure should disappear soon and be replaced by k8s one. So qserv/admin/templates will also disappear.

Reply:
I try to stick with that: https://12factor.net/config